Detections/Corelight/CorelightSMTPEmailSubjectNonAsciiCharacters.yaml

```diff
@@ -0,0 +1,34 @@
+id: 50c61708-9824-46f3-87cf-22490796fae2
+name: Corelight - SMTP Email containing NON Ascii Characters within the Subject
+description: |
+  'Detects where an emails contain non ascii characters within the Subject.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: Corelight
+    dataTypes:
+      - Corelight_CL
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1566 
+query: |
+  Corelight
+  | where EventType =~ 'smtp'
+  | where Subject startswith @'\=?utf-16'

```

Comment:
Test out hasprefix with this instead of startswith, the characters in it may causing matching issues, but I don't think they will and it will have better performance with hasprefix - https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/datatypes-string-operators

Reply:
updated